32_flask/templates/book.html

```diff
@@ -1,13 +1,14 @@
 <!DOCTYPE html>
```

Comment:
nit: use template inheritance to avoid code duplication

Reply:
ok